### PR TITLE
cloudapi: single uploadrequest

### PR DESCRIPTION
**Pull request by chloenayon**
Change imageRequest to accept a single uploadRequest instead of
an array of uploadRequests.

### CODE REVIEW

**Comment by Gundersanne** [comment]
Nice! Adding blocked until the related composer PR is merged (https://github.com/osbuild/osbuild-composer/pull/1267)

**Review by Gundersanne — APPROVED** [review]

**Comment by thozza** [comment]
Thanks for a quick rebase and merge @Gundersanne :partying_face:

**Pull request merged**
